cockatrice/src/carddatabasemodel.cpp

```diff
@@ -117,40 +117,78 @@ CardDatabaseDisplayModel::CardDatabaseDisplayModel(QObject *parent)
     setSortCaseSensitivity(Qt::CaseInsensitive);
 }
 
+
+bool CardDatabaseDisplayModel::lessThan(const QModelIndex &left, const QModelIndex &right) const {
+
+    QString leftString = sourceModel()->data(left).toString();
+    QString rightString = sourceModel()->data(right).toString();
+
+    if (leftString.compare(cardName, Qt::CaseInsensitive) == 0) {// exact match should be at top
```

Comment:
'æ' is 'ae' with card names. Im not sure its ever been used.
Not sure what you want me to do with that?